RealmSwift/App.swift

```diff
@@ -237,21 +237,51 @@ public extension App {
         }
     }
 
-    /// Login to a user for the Realm app.
-    /// @param credentials The credentials identifying the user.
-    /// @returns A publisher that eventually return `User` or `Error`.
+    /**
+    Updates the base url used by Atlas device sync, in case the need to roam between servers (cloud and/or edge server).
+     - parameter url: The new base url to connect to. Setting `nil` will reset the base url to the default url.
+     - note: Updating the base URL would trigger a client reset.
+     */
+    @preconcurrency
+    @available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+    @_spi(Private) func updateBaseUrl(to url: String?, _ completion: @Sendable @escaping (Result<Void, Error>) -> Void) {
+        self.__updateBaseURL(url) { error in
+            if let error = error {
+                completion(.failure(error))
+            } else {
+                completion(.success(()))
```

Comment:
There's a bunch of extra parentheses here.

Reply:
We are returning void in case of success

Reply:
Oh, I thought Result had an extra initializer when Success = Void but I guess that got dropped after the initial proposal.